connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoSinkOptions.scala

```diff
@@ -72,6 +74,10 @@ object KustoSinkOptions extends KustoOptions {
   // https://docs.microsoft.com/azure/data-explorer/kusto/management/auto-delete-policy
   // Use this option if you want to persist partial write results (as the failure could be of a single partition)
   val KUSTO_TEMP_TABLE_NAME: String = newOption("tempTableName")
+
+  // The batch size that we want to use for each "streaming batch". The default is 4MB.
+  val KUSTO_STREAMING_INGEST_SIZE = newOption("kustoStreamingIngestSize")
```

Comment:
Better to have the size in name - as in "kustoStreamingIngestSizeInMB" or something (I shoulde done it with the others when i introduced them..)  and the default for that would be "4" -